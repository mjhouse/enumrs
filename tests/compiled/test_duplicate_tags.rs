use enumrs::Tagged;

#[derive(Tagged)]
pub enum TestEnum {
    // Tag names should be unique for the
    // variant that they're on.
    #[tag(tagname)]
    #[tag(tagname)]
    Variant = 1,
}

pub fn main() {}