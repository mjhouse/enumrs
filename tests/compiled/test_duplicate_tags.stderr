error: Duplicate tag: 'tagname'
 --> tests/compiled/test_duplicate_tags.rs:8:5
  |
8 |     #[tag(tagname)]
  |     ^
